Makefile

```diff
@@ -1,5 +1,8 @@
 .PHONY: test docs
 
+COVERAGE_OPTS := --rcfile $(CURDIR)/.coveragerc
```

Comment:
again, since `.coveragerc` is the default file location i think coverage looks for it even if we don't specify. seems to work for me locally (even via tox). Does it fail for you if you drop the explicit `COVERAGE_OPTS`?